sharepoint/sharepoint-ps/sharepoint-online/Get-SPORestrictedAccessForSitesInsights.md

```diff
@@ -0,0 +1,244 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-sporestrictedaccessforsitesinsights
+applicable: SharePoint Online
+title: Get-SPORestrictedAccessForSitesInsights
+schema: 2.0.0
+author: itambakhe
+ms.author: itambakhe
+ms.reviewer:
+manager:
+---
+
+# Get-SPORestrictedAccessForSitesInsights
+
+## SYNOPSIS
+
+This cmdlet enables the administrator to check status of all active and available reports about insights on sites protected and access denials by Restricted access control policy when no report ID is present and to view or download the report if report ID is present.
+
+## SYNTAX
+
+```powershell
+Get-SPORestrictedAccessForSitesInsights -RACProtectedSites [-ReportId <Guid>] [-Action <ActionType>] [-InsightsSummary <Boolean>]
+```
+
+```powershell
+Get-SPORestrictedAccessForSitesInsights -ActionsBlockedByPolicy [-ReportId <Guid>] [-Content <ContentType>] [-Action <ActionType>]
+```
+
+## DESCRIPTION
+
+If this cmdlet is executed with `-RACProtectedSites` as parameter, it displays the status of all active and completed reports with the following properties:
+
+| Property             | Description                                                 |
+|:---------------------|:------------------------------------------------------------|
+| Id                   | The unique Id of the report.                                |
+| CreatedDateTimeInUtc | The date and time the report creation was triggered in UTC. |
+| Status               | The status of the report.                                   |
+
+If this cmdlet is executed with `-RACProtectedSites` `-ReportId` as parameter, top 100 sites with highest page views that are protected by Restrcited access control will be displayed with the following properties:
+
+| Property                     | Description                                                   |
+|:-----------------------------|:--------------------------------------------------------------|
+| SiteName                     | The name of the SharePoint site.                              |
+| SiteType                     | The type of the SharePoint site.                              |
+| SiteURL                      | The URL of the SharePoint site.                               |
+| SiteSensitivity              | The sensitivity label of the SharePoint site.                 |
+| PrimaryAdmin                 | Primary admin of the SharePoint Site                          |
+| BlockDownloadPolicy          | Boolean status of Block download policy on the site.          |
+| ConditionalAccessPolicy      | Authentication context set for the Conditional access policy  |
```

Comment:
>policy [](http://example.com/codeflow?start=87&length=6)

some sentences end with `.` while others don't -- please update so they're all consistent

Reply:
Made sure all the sentences end with ".". Also, this content is removed from the file now.